stingray/tests/test_crossspectrum.py

```diff
@@ -241,6 +242,9 @@ def __init__(self):
         with pytest.raises(AttributeError):
             lag = obj.time_lag()
 
+    def test_plot_simple(self):
+        self.cs.plot()
+        assert plt.fignum_exists(1)
```

Comment:
I don't know if it is possible, but can you give that figure a specific number and test for that specific number? Because I get the impression that this is testing for _any_ figure (1), produced by any other function.

Reply:
I just took `Lightcurve` plot test and used it here. It looks like this:
```
    def test_plot_simple(self):
        lc = Lightcurve(self.times, self.counts)
        lc.plot()
        assert plt.fignum_exists(1)
```

Reply:
That's also probably wrong for the same reason :)

Reply:
Please use this syntax, giving a unique label to the figure
https://github.com/matplotlib/matplotlib/blob/master/lib/matplotlib/tests/test_figure.py#L85